src/kOS.Safe/Execution/CPU.cs

```diff
@@ -1021,27 +1085,6 @@ public void RemoveTrigger(int triggerFunctionPointer)
             currentContext.RemoveTrigger(triggerFunctionPointer);
         }
 
-        /// <summary>
-        /// Put the CPU into wait state for now, which will reset next Update.
-        /// Also, if given a number of seconds, return the number of seconds into
-        /// the future that is, in game-terms, for the sake of making a timestamp
-        /// of when the wait is expected to be over.
-        /// </summary>
-        /// <param name="waitTime"></param>
-        /// <returns></returns>
-        public double StartWait(double waitTime)
-        {
-            if (currentStatus == Status.Running)
-                currentStatus = Status.Waiting;
-            return currentTime + waitTime;
-        }
-
-        public void EndWait()
-        {
-            if (currentStatus == Status.Waiting)
-                currentStatus = Status.Running;
-        }
-
```

Comment:
These are part of a way even older than the current develop branch way of doing this.  StartWait, for example, was never called from anywhere, and EndWait was only called internally from inside CPU, despite being public.  Anyway both have been replaced with new logic.